CharacterGenerator/Character.m

```diff
@@ -0,0 +1,67 @@
+//
+//  Character.m
+//  CharacterGenerator
+//
+//  Created by Henna on 8/2/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "Character.h"
+
+@implementation Character
+
+- (void) initializeName: (NSString *) name pushups: (int) pushupLimit fighter: (BOOL) canFight andBeer: (int) beerLimit withImage: (UIImage*) image;{
+    
+    self.name = name;
+    self.pushupLimit = pushupLimit;
+    self.canFight = canFight;
+    self.beerLimit = beerLimit;
+    self.superPower = self.determineSuperPower;
+    self.image = image;
+}
+
+- (NSString *) determineSuperPower {
```

Comment:
This series of nested if statements is really hard to read. Cuddling the `else`s would help. Consider another way of assigning superpowers.